pkg/cmd/ssh_aws.go

```diff
@@ -242,7 +246,7 @@ func (a *AwsInstanceAttribute) sshPortCheck() {
 		cmd := exec.Command("bash", "-c", ncCmd)
 		output, _ := cmd.CombinedOutput()
 		fmt.Println("=>", string(output))
-		if strings.Contains(string(output), "succeeded") {
+		if strings.Contains(string(output), "open") {
```

Comment:
This should be `succeeded`. 

With `open`, I get:

```
Bastion host instance running.
Opened SSH Port on Node.
Waiting 60 seconds for Bastion SSH port open
=> Connection to ... port 22 [tcp/*] succeeded!

=> Connection to ... port 22 [tcp/*] succeeded!

=> Connection to ... port 22 [tcp/*] succeeded!

=> Connection to ... port 22 [tcp/*] succeeded!

=> Connection to ... port 22 [tcp/*] succeeded!

=> Connection to ... port 22 [tcp/*] succeeded!

	SSH Port Open on Bastion TimeOut
(4/4) Cleanup
Cleaning up bastion host configurations...
```

With `succeeded`:
```
Bastion host instance running.
Opened SSH Port on Node.
Waiting 60 seconds for Bastion SSH port open
=> Connection to ... port 22 [tcp/*] succeeded!

Opened SSH Port on Bastion
SSH gardener@... => gardener@ip-...compute.internal
Warning: Permanently added '...' (ED25519) to the list of known hosts.
Warning: Permanently added 'ip-....compute.internal' (ED25519) to the list of known hosts.
(4/4) Cleanup
Cleaning up bastion host configurations...
```

Reply:
Hi @dansible , i used a newly introduced function `CheckIPPortReachable` which is using native go code (so we don't have dependency on different version of binary like your nc returns succeeded but mine returns open) to perform ssh port check